Source/scrollrt.cpp

```diff
@@ -53,6 +53,10 @@ int level_piece_id;
 DWORD sgdwCursWdt;
 void (*DrawPlrProc)(int, int, int, int, int, BYTE *, int, int, int, int);
 BYTE sgSaveBack[8192];
+static BYTE sgCursorBitmap[8192];
+static SDL_Surface *sgCursorSurface;
+static int sgCursorPaletteVersion, sgCursorWindowsResVer = -1;
+static int sgCursorCel = -1, sgCursorCol = -1;

```

Comment:
No idea, probably `color`. The naming stems from the variable `col` found later in the same file and is the original Diablo variable.

Reply:
> No idea, probably color.

Now that's scary. 

> The naming stems from the variable col found later in the same file and is the original Diablo variable.

If at all possible, let's try to avoid expanding on the problems from the original code. Either use a better name now, or consider just refactoring the old name to something better in a separate change.